src/App.tsx

```diff
@@ -15,48 +16,136 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortBy {
+  Alfabet = 'alphabet',
+  Length = 'length',
+  Indexdesc = 'reverse Default',
+  Default = 'Default',
+}
+
+const sortAlfabet = (a: string, b: string, direction: boolean) => {
+  return direction === true
```

Comment:
the direction parameter is already a boolean, you can directly use it in the ternary operator without explicitly comparing it to true